api/powershell/invite_members_to_org.ps1

```diff
@@ -0,0 +1,69 @@
+#Requires -version 7
+
+<#
+.SYNOPSIS
+Batch invite members to an organization
+
+.DESCRIPTION
+This script runs a batch organization invitation process for a given list of GitHub enterprise
+consumed licenses.
+
+The input is a CSV file with a column named "Handle or email", such as can be exported from the
+Enterprise settings > Enterprise licensing page. Users with appropriate permissions can export
+the CSV file, edit it in their favorite spreadsheet to select emails to invite, then use this
+script to invite them to an org.
+
```

Comment:
Minor nit: Maybe mention that this script is for "Enterprise Cloud" users (on github.com). Enterprise Server users would need a different API endpoint (`http(s)://[hostname]/api/v3`).

Reply:
Good call! I'll create a small PR with this suggestion.